### ISSUE TITLE
Allowing 'Humans' to play

### ISSUE THREAD

**Issue by R1j1t** [openai/gym#1035]
I used gym for a reinforcement learning project in my college. I was very impressed to see the diversity of environments gym offered. Now, having said that, while we were doing the project, we thought if it was possible for the human to interfere in the game. By interfere I mean, lets say at the start instead of agent taking random actions I as a player play the game and neural nets get trained while I play as well. 

The whole point of this was, we had a shortage of GPUs and so we couldn't train the agent completely. Also, the 2 games we trained on (cartpole-v1, Pong) were not very familiar with the panel members. So, this would also have served the purpose to give them an intuition of the game. 

I am no expert of Reinforcement learning and so this was what i thought and wanted to raise this as a feedback. Will be happy to hear from other of their viewpoint.

**Comment by tycho01** [comment]
looks like this may be [possible](https://github.com/openai/gym/blob/master/gym/utils/play.py#L26) already.

**Comment by R1j1t** [comment]
@tycho01 Thanks for the reply, I didn't knew that there was an option for the user to play as well. This is great, but still I this solves only half the problem, the other thing was to allow training during the player's plays and make gradients updates based on the rewards the players earned, which is not possible here. It doesn't return the `observation, reward, done, info`. We might need to add a one or two line there for that. 

But thanks for pointing out, I am glad they have it in here.

**Comment by tycho01** [comment]
agreed, that would be pretty interesting -- I do wonder if there are existing threads on that. I think OpenAI had written stuff about imitation/transfer learning, which may be relevant here. not sure if any of that has been incorporated into Gym though...

**Comment by tycho01** [comment]
it turns out there is also a [`keyboard_agent`](https://github.com/openai/gym/blob/master/examples/agents/keyboard_agent.py) script. :)
